crates/sui-json-rpc/src/api.rs

```diff
@@ -281,6 +284,20 @@ pub trait RpcFullNodeReadApi {
         /// the version of the queried object. If None, default to the latest known version
         version: SequenceNumber,
     ) -> RpcResult<GetPastObjectDataResponse>;
+
+    /// Return contents of a checkpoint, namely a list of execution digests
+    #[method(name = "getCheckpointContents")]
+    fn get_checkpoint_contents(
```

Comment:
Where can we get the CheckpointContentsDigest to call this method?

Reply:
Great point, I added a `get_checkpoint_summary` endpoint so that the content digest can be found there.